edn/src/edn.rustpeg

```diff
@@ -148,11 +148,11 @@ namespace_separator = "/"
 // TODO: Be more picky here
 // Keywords follow the rules of symbols, except they can (and must) begin with :
 // e.g. :fred or :my/fred. See https://github.com/edn-format/edn#keywords
-symbol_char_initial = [a-z] / [A-Z] / [0-9] / [*!_?$%&=<>]
-symbol_char_subsequent = [a-z] / [A-Z] / [0-9] / [-*!_?$%&=<>]
+symbol_char_initial = [a-z] / [A-Z] / [0-9] / [*!_?$%&=<>.]
+symbol_char_subsequent = [a-z] / [A-Z] / [0-9] / [-*!_?$%&=<>.]
 
-symbol_namespace = symbol_char_initial symbol_char_subsequent* (namespace_divider symbol_char_subsequent+)*
-symbol_name = ( symbol_char_initial+ / "." ) ( symbol_char_subsequent* / "." )
```

Comment:
I think the idea was related to "Symbols beginning or ending with . are reserved by Clojure.". That is: symbols are an alphanumeric character, followed by alphanum+some specials, and they can include a `java.divided.namespace`, an optional slash, and so on.

While Clojure's reader is very very lazy, and the `symbol` function doesn't do any validation, it's probably not *intentional* if either the EDN grammar or the Clojure reader allows a symbol like `...aaa......a.a..a...a.`

So we might have two pathways: 'normal' symbols, which must look like

`foo.bar_noo/b-az10`

and 'reserved' symbols, which can be things like

`...`, `.`, `.reserved/suffix-.`

That said, if Clojure reads it, and the EDN spec allows it, then I guess we have to allow variables in queries like `?..asda.as+das..aa.....`, right?